app/src/main/java/io/left/meshim/controllers/RightMeshController.java

```diff
@@ -347,12 +358,27 @@ private void handlePeerChanged(RightMeshEvent e) {
         if (message == null) {
             return null;
         }
-
-        MeshIMMessages.Message protoMsg = MeshIMMessages.Message.newBuilder()
-                .setMessage(message.getMessage())
-                .setTime(message.getDateAsTimestamp())
-                .build();
-
+        MeshIMMessages.Message protoMsg = null;
+        //if there is a filebyte to write in protobuf we write it.
+        if(!message.getFilePath().equals("")) {
+            try {
+                File file = new File(message.getFilePath());
+                protoMsg = MeshIMMessages.Message.newBuilder()
+                       .setMessage(message.getMessage())
+                       .setTime(message.getDateAsTimestamp())
+                       .setFileByte(ByteString.copyFrom(RightMeshController.getBytesFromFile(file)))
+                       .setFileExtension(message.getFileName())
+                       .build();
+            } catch (IOException e) {
+                e.printStackTrace();
```

Comment:
Is just ignoring this the best way to handle the exception? If so, can you document why?

What is the effect of not doing this? Should we communicate this exception somewhere else?